src/routes/routes.jsx

```diff
@@ -0,0 +1,22 @@
+import { Outlet, Routes, Route, BrowserRouter } from "react-router-dom";
+
+import React from "react";
```

Comment:
Fernando, poderia remover o Outlet e o React das importações e remover a linha entre eles.
```suggestion
import { Routes, Route, BrowserRouter } from "react-router-dom";
```